docs/sources/enterprise/aws-secrets-plugin.md

```diff
@@ -0,0 +1,87 @@
+---
+aliases:
+  - /docs/grafana/latest/enterprise/aws-secrets-plugin/
+description: 'Configuring the AWS Secrets Manager plugin'
+keywords:
+  - grafana
+  - secrets
+  - secretsmanager
+  - aws
+  - backend
+title: AWS Secrets Manager Integration
```

Comment:
```suggestion
title: Encrypt database secrets using AWS Secrets Manager
```

Reply:
this is kind of a long title. Any way to shorten it using `menuTitle`?